Lanchonete/Lanchonete/Model/Sandwich.swift

```diff
@@ -7,36 +7,41 @@
 
 import Foundation
 
-public enum SandwichType:  String, CaseIterable {
+public enum SandwichType:  String, CaseIterable, Equatable {
     
     case xBacon = "X-Bacon"
     case xEgg = "X-Egg"
-    case xBurguer = "X-Burguer"
+    case xBurger = "X-Burger"
     case xEggBacon = "X-EggBacon"
+    case xCustom = "X-Custom"
     
     var ingredientsInSandwich: [Ingredient] {
         switch self {
         case .xBacon:
             return [.hamburger,.cheese,.bacon]
-        case .xBurguer:
+        case .xBurger:
             return [.hamburger,.cheese]
         case .xEgg:
             return [.hamburger,.cheese,.egg]
         case .xEggBacon:
             return [.hamburger,.cheese,.egg,.bacon]
+        case .xCustom:
+            return []
         }
     }
     
     var name: String {
```

Comment:
ℹ️ The way `name` and `code` are being used, you could remove the `switch` blocks and do `return rawValue`, which will return the value set on the `enum` `case`.

Reply:
I agree with this, I changed it now. The class is a lot simpler